metricbeat/mb/module/wrapper.go

```diff
@@ -185,6 +186,11 @@ func (mw *Wrapper) Hash() uint64 {
 	return mw.configHash
 }
 
+// MetricSets return the list of metricsets of the module
+func (mw *Wrapper) MetricSets() []*metricSetWrapper {
```

Comment:
That was my first though toot, but then I would need to reimplement [fetch](https://github.com/elastic/beats/blob/master/metricbeat/mb/module/wrapper.go#L250), which is not a big deal but could cause issues when we add new metricset types, wdyt?